specs/_template/02_user_interface_android.rst

```diff
@@ -0,0 +1,32 @@
+=======  ==================  ======  ==========
+ epic          title         author   created
+=======  ==================  ======  ==========
+`EPIC`_  SPECIFICATION_NAME  AUTHOR  CREATED_AT
```

Comment:
EPIC should just be the number, right?

Reply:
the intention was to provide this label so that you could search and replace it in the doc.